tools/commentMonitor/Dockerfile

```diff
@@ -1,16 +1,6 @@
-# NOTE: building this requires it to have the build context of the repository root
-FROM golang:1.12-alpine AS builder
+FROM quay.io/prometheus/busybox:latest
+LABEL maintainer="The Prometheus Authors <prometheus-developers@googlegroups.com>"
 
```

Comment:
`commentMonitor` and `amGithubNotifier` are built successfully when using multi-staged builds, is there any specific reason to revert this other than to have a consistent way to build every tool? @krasi-georgiev suggested we should move towards multi-staged builds, so I think the ones that are working can be kept that way if possible? otherwise, we add this change.

cc: @krasi-georgiev

Reply:
What is a multi-staged build? `store_artifact` them, then use it later ? Seems it only applies to `scaler`, `fake-webserver` and `prombench`.

Reply:
no multistage is for the docker builds. Basically keeps the final docker image very small because it includes only the binary

https://docs.docker.com/develop/develop-images/multistage-build/

Reply:
@simonpasquier suggested using `quay.io/prometheus/busybox:latest` in #218. (It contains some root CAs, so we don't need to copy them when building`fake-webserver`.)

I think the size of image is mainly affected by the binary size and base image size.  Compare the image size of my latest builds docker.io/nevill/{prombench, commentmonitor, amgithubnotifier, fake-webserver, scaler, load-generator} with docker.io/prombench/{prombench, commentmonitor, amgithubnotifier, fake-webserver, scaler, load-generator}, they're almost the same.

Reply:
There are two jobs in this circle CI config. The first is `build_and_test`, which will create the binaries, and cache them (by calling `persist_to_workspace`). In the next job `publish`, we just need to COPY the binaries from cache into the image under constructing. That's why every `go build` is removed in `Dockerfile`.

Reply:
This will remove the posibility to build the docker images locally which is usefull sometimes.

`GO111MODULE=on go build -a -tags netgo -o /go/bin/commentMonitor` would cause downlading the source files of all dependancies so even if doesn't make big difference now it is better to keep the multistage builds. Ofcourse if there is no good reason to have it as a single stage.

> @simonpasquier suggested using quay.io/prometheus/busybox:latest in #218. (It contains some root CAs, so we don't need to copy them when buildingfake-webserver.)

that is a good suggestion so should use it if possible.